testing/bdd/features/styles/ebuttd_style_references.feature

```diff
@@ -0,0 +1,89 @@
+@styles @document @ebuttd_conversion 
+Feature: Remove style elements that refer to other style elements
+
+  Examples:
+    | xml_file                     |
+    | style_element_references.xml |
+
+
+  Scenario: Convert style element with reference to another style to a presentationally equivalent one
+    Given an xml file <xml_file>
+    When  it contains style "s_default"
+    And   style "s_default" has attribute "color" set to "white"
+    And   style "s_default" has attribute "backgroundColor" set to "black"
+    And   it contains style "s_font_reith"
+    And   style "s_font_reith" has attribute "style" set to "s_default"
+    And   style "s_font_reith" has attribute "fontFamily" set to "reith"
+    And   it contains style "s_font_monospace"
+    And   style "s_font_monospace" has attribute "fontFamily" set to "monospace"
+    And   it contains style "s_top"
+    And   style "s_top" has attribute "style" set to "s_font_reith s_font_monospace"
+    And   style "s_top" has attribute "color" set to "yellow"
+    And   it contains some text with style "s_top"
+    When  the document is generated
+    And   the EBU-TT-Live document is converted to EBU-TT-D
+    Then  the ebu_tt_d document contains style "s_top" with attribute "fontFamily" set to "monospace"
+    And   the ebu_tt_d document contains style "s_top" with attribute "color" set to "#ffff00ff"
```

Comment:
no change needed here, but just for posterity, if the code returned `color` set to `#ffff00` or `backgroundColor` set to `#000000` that would be equally valid.

Reply:
I mean I _could_ get it to check for either #ffff00ff or #ffff00 but that's probably going to be more hassle than it's worth :P

Reply:
Agreed.